aphrodite/aphrodite.d.ts

```diff
@@ -69,6 +70,30 @@ declare module "aphrodite" {
     }
 
     export var StyleSheetTestUtils: StyleSheetTestUtilsStatic;
+
+    export interface SelectorHandler {
+        (selector: string, baseSelector: string, callback: (selector: string) => string): string;
+    }
+
+    export interface Extension {
+        selectorHandler?: SelectorHandler,
```

Comment:
why is `selectorHandler` optional? Does `extend` skip objects without that property?